src/constants.js

```diff
@@ -34,6 +34,11 @@ exports.STU3 = {
 	}
 };
 
+exports.CONFIG_KEYS = {
+	OBSERVATION: 'observation',
```

Comment:
Do we need to add keys here for every resource we create?

Reply:
Yes, we need a mapping from the user provided config to a resource. If you look at node-fhir-server-mongo or the runner file in this repo. You can see an example of where these are used. E.g.

```javascript
injected_config = {
  profiles: {
    dstu2: {
      patient: { ...patient config },
      observation: { ...observation config }
    }
  }
}
```